### DISCUSSION TITLE
Facebook strips away Nuxt 3 query array params

### DISCUSSION THREAD

**Discussion by madsh93** [nuxt/nuxt#20455] [Questions]
Hi

I've been advertising on Facebook and have been having issues with the way Facebook handles query strings.

This issue also occurs on Facebook Messenger.

E.g. sending this url on Facebook Messenger or through ads:

https://www.domain.com/checkout?product_id=122&leasing_period=60&kilometers_per_year=15000&color_id=106&equipment_ids=59&equipment_ids=62&deal_type=leasing

(Note there are 2 equipment ids)

Will result in:

https://www.domain.com/checkout?product_id=122&leasing_period=60&kilometers_per_year=15000&color_id=106&equipment_ids=62&deal_type=leasing

(Note there is only 1 equipment id)

It should also be noted that this doesn't happen always. Only sometimes. But it's not good when it happens on ads as it'll cost money.

But clearly Facebook has troubles understanding the way Nuxt 3 uses arrays in query params.

Has anybody else experienced this?

**Comment by warflash** [comment]
It's weird to imagine nuxt itself would sanitize the query params tbh (not impossible though). Also weird it only happens sometimes. Are you sure it's not to do how you're parsing the incoming params in your nuxt app?
If you find a way to reproduce it that might help with looking into the issue

**Reply by madsh93** [reply]
I'm not really sure how to reproduce. I can't be on my end, cause if I literally right-click copy link on Facebook it is already stripped away.

**Reply by madsh93** [reply]
If you have Facebook. Try sending this to someone:

https://www.domain.com/demo?equipment_ids=26&equipment_ids=28

 You will see the link strips away the first equipment_ids.
